src/main/scala/com/datastax/sparkstress/ReadTask.scala

```diff
@@ -11,21 +11,28 @@ import org.joda.time.DateTime
 
 object ReadTask {
   val ValidTasks = Set(
-    "ftsallcolumns",
-    "ftsfivecolumns",
-    "ftsonecolumn",
-    "ftspdclusteringallcolumns",
-    "ftspdclusteringfivecolumns",
-    "jwcallcolumns",
-    "jwcpdclusteringallcolumns",
-    "jwcrpallcolumns",
-    "pdcount",
-    "retrievesinglepartition"
+    "FTSAllColumns",
+    "FTSFiveColumns",
+    "FTSOneColumn",
+    "FTSPDClusteringAllColumns",
+    "FTSPDClusteringFiveColumns",
+    "JWCAllColumns",
+    "JWCPDClusteringAllColumns",
+    "JWCRPAllColumns",
+    "PDCount",
+    "FTSOneColumn_DS",
+    "FTSTwoColumns_DS",
+    "FTSThreeColumns_DS",
+    "FTSFourColumns_DS",
```

Comment:
instead of having new tests do would it make sense to just have a flag "--datasets?" or maybe given that this will be the main method going forwardss "--rdd" and the default is the dataset methods?

Reply:
Yeah that makes sense. I'll add some safety checks too, to protect a user from attempting invalid read requests like `FTSPDClusteringAllColumns + Datasets + from DSEFS` for example.